gneiss/plot/_heatmap.py

```diff
@@ -0,0 +1,124 @@
+"""
+Plotting functions (:mod:`gneiss.plot`)
+===============================================
+
+.. currentmodule:: gneiss.plot
+
+This module contains plotting functionality
+
+Functions
+---------
+
+.. autosummary::
+   :toctree: generated/
+
+   heatmap
+
+"""
+
+import numpy as np
+from ete3 import TreeStyle, AttrFace, ProfileFace
+from ete3 import ClusterNode
+from ete3.treeview.faces import add_face_to_node
+import io
```

Comment:
I guess `import io` should better be before `import numpy`, since `io` is a built-in package of Python.